### PR TITLE
Update of the FAQ Log entry

### PR DESCRIPTION
**Pull request by vogella**
Lets promote ILog.get() instead Platform.getLog():

### CODE REVIEW

**Comment by vogella** [comment]
@HannesWell WDTY? 

@thahnen FYI, we like to see more PR from you (also code related if you find the time).

File: docs/FAQ/FAQ_How_do_I_use_the_platform_logging_facility.md [hunk 1]
```diff
@@ -1,44 +1,47 @@
...
       ...
       private static final Bundle BUNDLE = FrameworkUtil.getBundle(MyLogger.class);
-      private static final ILog LOGGER = Platform.getLog(BUNDLE);
+      
+   
       ...
       public static void test(){
-         try {
-            log("Eclipse Style Logging");
+		 ILog.get().info("Starting");
```
Comment (merks) [inline docs/FAQ/FAQ_How_do_I_use_the_platform_logging_facility.md:20]: Maybe it's not a good idea to use tabs.

Comment (vogella) [reply docs/FAQ/FAQ_How_do_I_use_the_platform_logging_facility.md]: Should be resolved

File: docs/FAQ/FAQ_How_do_I_use_the_platform_logging_facility.md [hunk 2]
```diff
@@ -1,44 +1,44 @@
...
+      
+   
       ...
       public static void test(){
+         ILog.get().info("Starting");
          try {
-            log("Eclipse Style Logging");
+            // do something that may cause an exception
          }
          catch(Exception e){
```
Comment (merks) [inline docs/FAQ/FAQ_How_do_I_use_the_platform_logging_facility.md:24]: Nit.

```suggestion
         catch (Exception e){
```

Comment (vogella) [reply docs/FAQ/FAQ_How_do_I_use_the_platform_logging_facility.md]: Changed

**Comment by vogella** [comment]
Ignoring freeze as this is only online documentation

File: docs/FAQ/FAQ_How_do_I_use_the_platform_logging_facility.md [hunk 3]
```diff
@@ -1,44 +1,44 @@
...
 Each plug-in has its own log associated with it, but all logged information eventually makes its way into the platform log file (see the getLogFileLocation method on Platform).
 
-You can write any kind of IStatus object to the log file, including a MultiStatus if you have hierarchies of information to display. If you create your own subclass of the utility class Status, you can override the getMessage method to return extra information to be displayed in the log file. Many plug-ins add utility classes for writing messages and errors to the log:
-
-      import org.eclipse.core.runtime.Status;
+      import org.eclipse.core.runtime.ILog;
       ...
       public class MyLogger {
       ...
       private static final Bundle BUNDLE = FrameworkUtil.getBundle(MyLogger.class);
```
Comment (HannesWell) [inline docs/FAQ/FAQ_How_do_I_use_the_platform_logging_facility.md:15]: Is this line still necessary? 
At least while viewing this on the phone I can't see a reference.

**Review by HannesWell — APPROVED** [review]
Thanks. I personally also prefer the new code.

I just wonder if the example should be put in a markdown code block wrapped in three back-ticks.

**Pull request merged**